zulip/integrations/confluence/confluence

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Confluence integration for Zulip
+#
+# Copyright © 2014 Zulip, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+import os
+import sys
+import json
+import requests
+import argparse
+from six.moves.configparser import ConfigParser, NoSectionError, NoOptionError
+import zulip
+
+CONFIGFILE = os.path.expanduser("~/.zulip_confluencerc")
+
+INSTRUCTIONS = r"""
+confluence --config-file=~/.zuliprc --frequency 5
+Send Confluence pages to a Zulip stream.
+
+To use this script:
+1. Set up Confluence authentication, as described below
+2. Set up a Zulip bot user and download its `.zuliprc`
+   config file to e.g. `~/.zuliprc`
+3. Subscribe the bot to the stream that will receive Confluence updates
+4. Configure a crontab entry for this script. A sample crontab entry
+that will process spaces and pages every 5 minutes is:
+*/5 * * * * /usr/local/share/zulip/integrations/confluence/confluence [options]
+
+== Setting up Confluence authentications ==
+
+Run this on a personal or trusted machine, because your API key is
+visible to local users through the command line or config file.
+
+Please create a ~/.zulip_confluencerc with the following contents:
+
+[Confluence]
+username =
+token =
+server_base_url =
+
+To obtain an API token from your Atlassian account:
+
+1. Log in to https://id.atlassian.com.
+2. Click API token.
+3. From the dialog that appears, enter a memorable and concise Label for your token and click Create.
+4. Click Copy to clipboard, then paste the token in .zulip_confluencerc:
+
+"""
```

Comment:
Anything that needs to be clicked or is a UI element should be bolded. :)

Reply:
Actually, ignore this one, for some reason, I thought this was a Markdown file haha.